carbonmark/components/pages/Project/index.tsx

```diff
@@ -45,6 +48,15 @@ export type PageProps = {
 
 const Page: NextPage<PageProps> = (props) => {
   const { data: project } = useGetProjectsId(props.projectID);
+  const { data: activities } = useGetProjectsIdActivity(props.projectID, {
```

Comment:
Done in 29fd4a049a3726b46ce178b2f9004ac1f10547e3

The preloading for the project data was broken too because of the way the SDK generates SWR keys vs how they were generated before. Please check.

It must be broken in other pages too. But maybe we can make a global check in a separate PR.

Reply:
Yea unfortunately this was an unexpected result of Kubb. Technically the preloading wasnt working anyway because we weren't leveraging swr on any of our calls before anyway. Will fix asap